pandas/tests/io/json/test_pandas.py

```diff
@@ -42,20 +42,10 @@ def assert_json_roundtrip_equal(result, expected, orient):
 
 @pytest.mark.filterwarnings("ignore:the 'numpy' keyword is deprecated:FutureWarning")
 class TestPandasContainer:
-    @pytest.fixture(scope="function", autouse=True)
-    def setup(self, datapath):
-        self.dirpath = datapath("io", "json", "data")
+    @pytest.fixture(autouse=True)
+    def setup(self, datapath, monkeypatch):
```

Comment:
what? why would you need to chdir?

Reply:
We use this same idiom in the excel file tests to read in files directly from where they are stored. Can alternately use datapath directly in the tests at the cost of more code duplication

Reply:
this is a really strange pattern, would prefer to clean it up

Reply:
i find this a really strange pattern which is not standard and likely not even needed

pls fix